indexing-service/src/test/java/org/apache/druid/indexing/common/task/concurrent/CommandQueueTask.java

```diff
@@ -133,13 +133,19 @@ private <V> void addToQueue(Command<V> command)
   private <V> V waitForCommandToFinish(Command<V> command)
   {
     try {
-      return command.value.get(10, TimeUnit.SECONDS);
+      return command.value.get(1000, TimeUnit.SECONDS);
```

Comment:
This is a very long timeout for tests. Is this really needed?

Reply:
I had forgotten to revert it after debugging.